include/factory_data.h

```diff
@@ -28,10 +28,12 @@
  * Do not access fields directly. Use API functions only. */
 typedef struct {
   uint32_t _hardware;
-  uint32_t _serial_number;
+  uint8_t _mfg_serial_number[17];
+  uint8_t _uuid[16];
   uint32_t _timestamp;
   uint8_t  _nap_key[16];
   uint8_t  _mac_address[6];
+  uint32_t _factory_stage;
```

Comment:
The plan was to update this for the two stages of testing, and after a customer return. The factory data would be erased and rewritten after each of these stages.